PHPCI/Builder.php

```diff
@@ -188,6 +188,14 @@ public function execute()
         $this->build->sendStatusPostback();
         $success = true;
 
+        $previous_build = $this->build->getProject()->getPreviousBuild($this->build->getBranch());
+
+        if ($previous_build) {
+            $previous_state = $previous_build->getStatus();
+        } else {
+            $previous_state = Build::STATUS_NEW;
+        }
```

Comment:
Recommend moving else section to above the if section as default behavior. Else should never be necessary. 

```php
        $previous_state = Build::STATUS_NEW;

        if ($previous_build) {
            $previous_state = $previous_build->getStatus();
        }
```